packages/integration-tests/tests/sign-in-experience.test.ts

```diff
@@ -36,4 +46,37 @@ describe('admin console sign-in experience', () => {
 
     expect(updatedSignInExperience).toMatchObject(newSignInExperience);
   });
+
+  it('should be able to setup sign in methods after connectors are enabled', async () => {
+    // Setup a social connector
+    await updateConnectorConfig(facebookConnectorId, facebookConnectorConfig);
+    await enableConnector(facebookConnectorId);
+
+    // Setup a SMS connector
+    await updateConnectorConfig(twilioSmsConnectorId, twilioSmsConnectorConfig);
+    await enableConnector(twilioSmsConnectorId);
+
+    // Setup an email connector
+    await updateConnectorConfig(sendgridEmailConnectorId, sendgridEmailConnectorConfig);
+    await enableConnector(sendgridEmailConnectorId);
+
+    // Set up sign-in methods
+    const newSignInMethods: Partial<SignInMethods> = {
+      username: SignInMethodState.Primary,
+      sms: SignInMethodState.Secondary,
+      email: SignInMethodState.Secondary,
+      social: SignInMethodState.Secondary,
+    };
+
+    const updatedSignInExperience = await authedAdminApi
```

Comment:
will this API check connector validity?

Reply:
This API will not, but the connector will be checked on setup.